### DISCUSSION TITLE
Example Podman test.pod with Quadlet doesn't want to generate after daemon-reload

### DISCUSSION THREAD

**Discussion by PrivateN00b** [containers/podman#23464] [Q&A] [answered]
Distro: Rocky Linux 9
Kernel version: 5.14.0-427.24.1.el9_4.x86_64
Podman version: 4.9.4-rhel

I'm trying to generate a pod via the Quadlet method, but somehow no matter how I try it just doesn't generate a pod (empty response from podman pod ls). I've named it test.pod in the .config/containers/systemd/ directory. I've tried changing the PodName, even not using this attribute, adding a network to it to no avail.

Here is the example pod: [https://docs.podman.io/en/latest/mar....html#examples](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#examples)

[Pod]
PodName=test

What logs should I provide?

**Comment by Luap99** [comment, marked as answer]
pod units were added in podman 5.0
